README.md

```diff
@@ -15,7 +22,6 @@ Build instructions: Android Studio IDE
 - In Android Studio, go to menu File -> Import Project
 - In the dialog box "Select Gradle Project Import", find this project's root directory in your filesystem and click 'OK'
 
-Screenshots
----------
+# Screenshots
```

Comment:
Given that those screenshots are already in the tutorial, do you think they provide relevant information here?

Reply:
Nope, removed them :P

Reply:
👍